pkg/gameservers/gameservers_test.go

```diff
@@ -47,14 +48,17 @@ func TestAddress(t *testing.T) {
 	fixture := map[string]struct {
 		node            *corev1.Node
 		expectedAddress string
+		externalDNS     bool
 	}{
 		"node with external ip": {
 			node:            &corev1.Node{ObjectMeta: metav1.ObjectMeta{Name: nodeFixtureName}, Status: corev1.NodeStatus{Addresses: []corev1.NodeAddress{{Address: "12.12.12.12", Type: corev1.NodeExternalIP}}}},
 			expectedAddress: "12.12.12.12",
+			externalDNS:     runtime.FeatureEnabled(runtime.NodeExternalDNS),
```

Comment:
This is a great first start, but rather than passing through, we can change the feature flag on the fly.

[here is a good example](https://github.com/googleforgames/agones/blob/master/pkg/apis/agones/v1/gameserver_test.go#L131) of where we change the feature flag on during testing, but also lock the test so that it doesn't interfere with parallel running tests.